lib/salus/processor.rb

```diff
@@ -93,42 +95,84 @@ def fetch_config_file(source_uri, repo_path)
 
     def scan_project
```

Comment:
method 'def scan_project' has become long. Is it possible to extract some part of it to a separate method?